m.neural_network.preparedata.worker_nullcells/m.neural_network.preparedata.worker_nullcells.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+"""############################################################################
+#
+# MODULE:      m.neural_network.preparedata.worker_nullcells
+# AUTHOR(S):   Guido Riembauer, Anika Weinmann
+# PURPOSE:     Worker module for m.neural_network.preparedata to check null
+#              cells
+# COPYRIGHT:   (C) 2024 by mundialis GmbH & Co. KG and the GRASS Development
+#              Team.
+#
+# 		This program is free software under the GNU General Public
+# 		License (v3). Read the file COPYING that comes with GRASS
+# 		for details.
+#
+#############################################################################
+"""
+
+# %Module
+# % description: Worker module for m.neural_network.preparedata to check null cells.
+# % keyword: raster
+# % keyword: statistics
+# %end
+
+# %option
+# % key: n
+# % type: string
+# % required: no
+# % multiple: no
+# % key_desc: value
+# % description: Value for the northern edge
+# % guisection: Bounds
+# %end
+
+# %option
+# % key: s
+# % type: string
+# % required: no
+# % multiple: no
+# % key_desc: value
+# % description: Value for the southern edge
+# % guisection: Bounds
+# %end
+
+# %option
+# % key: e
+# % type: string
+# % required: no
+# % multiple: no
+# % key_desc: value
+# % description: Value for the eastern edge
+# % guisection: Bounds
+# %end
+
+# %option
+# % key: w
+# % type: string
+# % required: no
+# % multiple: no
+# % key_desc: value
+# % description: Value for the western edge
+# % guisection: Bounds
+# %end
+
+# %option
+# % key: res
+# % type: string
+# % required: no
+# % multiple: no
+# % key_desc: value
+# % description: 2D grid resolution (north-south and east-west)
+# % guisection: Resolution
+# %end
+
+# %option G_OPT_R_INPUT
+# % key: map
+# % label: The name of input raster map
+# % guisection: Input
+# %end
+
+# %option
+# % key: tile_name
+# % type: string
+# % required: yes
+# % multiple: no
+# % key_desc: name
+# % label: Unique Name of the tile
+# %end
+
+# %option
+# % key: new_mapset
+# % type: string
+# % required: yes
+# % multiple: no
+# % label: Name for new mapset
+# %end
+
+import os
+import sys
+
+import grass.script as grass
+from grass_gis_helpers.mapset import switch_to_new_mapset
+
+
```

Comment:
It would be best to have the move back to the original mapset in a cleanup function, such that it also happens in case of an error, see also: https://github.com/mundialis/m.neural_network/pull/2#discussion_r1891181678